_pages/release-notes.md

```diff
@@ -10,79 +10,84 @@ tags: [alg-dev-getting-started, app-dev-getting-started]
 title:  "Release Notes"
 ---
 
-## Third-Party Secret Provider Plugins
+## Q2 2021 Release Notes ##
 
-We’re excited to announce that we now have documented example implementations of secret provider plugins for Hashicorp Vault and Azure Key Vault. This enables Algorithmia Enterprise customers to use their existing accounts in these third-party vaulting systems to store and manage their sensitive data, and for their Algorithmia algorithms to access those secrets at run time. To use this feature, a cluster administrator must first configure the new secret provider by uploading the secret provider plugin module through the Secret Store page in the admin panel.
+The following features are available in Algorithmia LTS version 21.0.
+{: .notice-info}
+
+### Third-Party Secret Provider Plugins ###
+
+We’re excited to announce that we now have documented example implementations of [secret provider plugins](https://training.algorithmia.com/exploring-the-admin-panel/842511#secret-provider-plugins) for Hashicorp Vault and Azure Key Vault. This enables Algorithmia Enterprise customers to use their existing accounts in these third-party vaulting systems to store and manage their sensitive data, and for their Algorithmia algorithms to access those secrets at run time. To use this feature, a cluster administrator must first configure the new secret provider by uploading the secret provider plugin module through the Secret Store page in the admin panel.
 
 ---
 
-## Algorithm Execution Errors
+### Algorithm Execution Errors ###
 
 Non-admin users can now view algorithm execution errors from the last seven days for specific algorithms or from all algorithms they’ve called. These error logs can also be retrieved through the Algorithmia API.
 
 All errors are captured and included in the error log, allowing users to debug with a single glance. Errors from algorithms called in a pipeline are bubbled up to the top-level calling algorithm. Error details include timestamp, error type, execution metadata, stacktrace, and partial algorithm input.
 
 This feature is disabled by default and can be enabled globally on an Algorithmia cluster by a cluster administrator.
 
-### Algorithm-specific errors
+#### Algorithm-specific errors ####
 
-Each algorithm’s execution errors are displayed on the algorithm’s profile page in a dedicated tab. Errors are only visible to users who are viewing an algorithm they own or that’s owned by an organization of which they’re a member.
+Each algorithm’s execution errors are displayed on the [algorithm’s profile page]({{site.baseurl}}/platform/algorithm-profile#errors) in a dedicated tab. Errors are only visible to users who are viewing an algorithm they own or that’s owned by an organization of which they’re a member.
```

Comment:
do we always use this format? I assumed that a relative path would work. e.g. `/developer/platform/algorithm-profile#error`

Reply:
It's sort of a mix throughout the docs, haha, and I know that works so I use it. I will try the hard-coded version in the future for simplicity.